sandbox/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+const fs = require('fs')
+
+const path = require('path')
+
+const express = require('express')
+
+const webpackDevMiddleware = require('webpack-dev-middleware')
+
+const webpack = require('webpack')
+
+const { CONFIG, logger } = require('./util')
+
+const webpackConfig = require('./webpack.config.dev')
+
+const livereloadMiddleware = require('./livereload/middleware')
+
+const compiler = webpack(webpackConfig)
+
+const devMiddleware = webpackDevMiddleware(compiler, webpackConfig.devServer)
+
+const indexHtml = fs.readFileSync(
+  path.resolve(__dirname, './public/index.html'),
+  'utf-8' // eslint-disable-line comma-dangle
+)
+
+const app = express()
+
+const server = require('http').Server(app)
+
+app.use(devMiddleware)
+
+app.use(livereloadMiddleware(compiler, server))
```

Comment:
It'd be great if we could set up hot reloading (in a future PR please)

This would allow us to test hot reloading weirdness, which is unfortunately quite relevant right now